etc/poco/README.md

```diff
@@ -37,18 +37,20 @@ Refer to [Testing](https://developer.atlassian.com/platform/poco/policies/workfl
 Poco bundles can be manually published to dev and staging environments using atlas-cli. Note that this will apply
 to the deployment stack already in service without needing to redeploy.
 
+### Publish into dev
 ```shell
```

Comment:
Separated these out so you can use the copy button without getting a bunch of stuff you don't need to dev/stg